cmake/DD4hepCPack.cmake

```
@@ -0,0 +1,35 @@
+
+set(CPACK_PACKAGE_NAME "dd4hep")
+set(CPACK_SET_DESTDIR TRUE)
+set(CPACK_PACKAGE_HOMEPAGE_URL "https://github.com/AIDASoft/dd4hep")
```

Is this meant as homepage or repo page? Otherwise
```suggestion
set(CPACK_PACKAGE_HOMEPAGE_URL "https://github.com/AIDASoft/dd4hep")
set(CPACK_PACKAGE_HOMEPAGE_URL "https://dd4hep.cern.ch")
```

---

I don't think it matters too much, however that particular URL gives me a `SSL_ERROR_BAD_CERT_DOMAIN `. I'll change it to `https://cern.ch/dd4hep`

---

I still have to create the https openshift for it